lib/model/rwfolder.go

```diff
@@ -1397,15 +1412,8 @@ func (f *sendReceiveFolder) performFinish(state *sharedPullerState) error {
 			fallthrough
 		// Permissions have changed, means the file has changed, rescan.
 		case !f.ignorePermissions(state.curFile) && state.curFile.HasPermissionBits() && !scanner.PermsEqual(state.curFile.Permissions, curMode):
-			l.Debugln("file modified but not rescanned; not finishing:", state.curFile.Name)
-			// Scan() is synchronous (i.e. blocks until the scan is
-			// completed and returns an error), but a scan can't happen
-			// while we're in the puller routine. Request the scan in the
-			// background and it'll be handled when the current pulling
-			// sweep is complete. As we do retries, we'll queue the scan
-			// for this file up to ten times, but the last nine of those
-			// scans will be cheap...
-			go f.Scan([]string{state.curFile.Name})
+			l.Debugln("file unknow or modified but not scanned; not finishing:", state.curFile.Name)
```

Comment:
typo in "unknown". It's in a debug message so I would hardly care except someone else is going to find it during some automated scan and file a separate pull request on it which I'll then have to handle